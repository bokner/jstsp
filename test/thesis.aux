\relax 
\providecommand\hyper@newdestlabel[2]{}
\catcode `"\active 
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\babel@aux{russian}{}
\@writefile{toc}{\contentsline {section}{\numberline {1}\IeC {\CYRP }\IeC {\cyro }\IeC {\cyrs }\IeC {\cyrt }\IeC {\cyra }\IeC {\cyrn }\IeC {\cyro }\IeC {\cyrv }\IeC {\cyrk }\IeC {\cyra } \IeC {\cyrz }\IeC {\cyra }\IeC {\cyrd }\IeC {\cyra }\IeC {\cyrch }\IeC {\cyri }}{1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}\IeC {\CYRP }\IeC {\cyrr }\IeC {\cyri }\IeC {\cyrm }\IeC {\cyre }\IeC {\cyrr }}{1}{section.2}}
\@writefile{toc}{\contentsline {section}{\numberline {3}(}{1}{section.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}How to add Comments}{1}{subsection.3.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}How to add Tables}{1}{subsection.3.2}}
\citation{greenwade93}
\bibstyle{alpha}
\bibdata{sample}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces An example table.}}{2}{table.1}}
\newlabel{tab:widgets}{{1}{2}{An example table}{table.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.3}How to write Mathematics}{2}{subsection.3.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.4}How to create Sections and Subsections}{2}{subsection.3.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.5}How to add Lists}{2}{subsection.3.5}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.6}How to add Citations and a References List}{2}{subsection.3.6}}
