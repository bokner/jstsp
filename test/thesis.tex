\documentclass[a4paper]{article}

%% Language and font encodings
\usepackage[english,russian]{babel}
\usepackage[utf8x]{inputenc}
\usepackage[T1]{fontenc}

%% Sets page size and margins
\usepackage[a4paper,top=3cm,bottom=2cm,left=3cm,right=3cm,marginparwidth=1.75cm]{geometry}

%% Useful packages
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage[colorinlistoftodos]{todonotes}
\usepackage[colorlinks=true, allcolors=blue]{hyperref}

\title{Вычислительные эксперименты по решению задачи упорядочения работ и перезагрузки приспособлений (Job Sequencing and Tool Switching Problem - JSTSP)}
\author{Борис Окнер}

\begin{document}
\maketitle

\begin{abstract}

\end{abstract}

\section{Постановка задачи}

Постановка основной задачи (JSTSP). Задано множество приспособлений Т и множество работ J = {1,...,n}. Для каждой работы j  J задан набор приспособлений Тj. Магазин приспособлений C может содержать минимальное количество приспособлений равное максимальному количеству приспособлений 
$
\|T_{max}|  = max{|T_j| : j \in J} 
$
необходимому для выполнения одной работы, но строго меньше чем |T|. Т.е. емкость магазина недостаточна для размещения всех приспособлений из Т,  т.е. 
$
\|T_{max}| <= |C| < |T|
$
Нужно найти такую последовательность выполнения всех работ, чтобы суммарное количество перезагрузок приспособлений было минимальным.

\section{Пример}
Таблица 1. 6 работ, 9 приспособлений, емкость магазина - 4.

\begin{tabular}{ c c c c c c c}
 Работы & 1 & 2 & 3 & 4 & 5 & 6 \\ 
 Приспособления & 1 & 1 & 2 & 1 & 3 & 1 \\  
  & 4 & 3 & 6 & 5 & 5 & 2 \\
  & 8 & 5 & 7 & 7 & 8 & 4 \\
  & 9 &  & 8 & 9 &  &  \\
\end{tabular}

Таблица 2. Оптимальное решение для задачи, описанной Таблицей 1.

\begin{tabular}{ c c c c c c c}
 Работы & 4 & 2 & 5 & 3 & 6 & 1 \\ 
 Содержание магазина & 1 & 1 & 8 & 8 & 8 & 8 \\ 
  & 5 & 5 & 5 & 6 & 1 & 1 \\
  & 7 & 7 & 7 & 7 & 4 & 4 \\
  & 9 & 3 & 3 & 2 & 2 & 9 \\
 Количество перезагрузок & 0 & 1 & 1 & 2 & 2 & 1 \\ 
\end{tabular}

\section(Список литературы)

\subsection{How to add Comments}

Comments can be added to your project by clicking on the comment icon in the toolbar above. % * <john.hammersley@gmail.com> 2014-09-03T09:54:16.211Z:
%
% Here's an example comment!
%
a comment, simply click the reply button in the lower right corner of the comment, and you can close them when you're done.


\subsection{How to add Tables}

Use the table and tabular commands for basic tables --- see Table~\ref{tab:widgets}, for example. 

\begin{table}
\centering
\begin{tabular}{l|r}
Item & Quantity \\\hline
Widgets & 42 \\
Gadgets & 13
\end{tabular}
\caption{\label{tab:widgets}An example table.}
\end{table}

\subsection{How to write Mathematics}

\LaTeX{} is great at typesetting mathematics. Let $X_1, X_2, \ldots, X_n$ be a sequence of independent and identically distributed random variables with $\text{E}[X_i] = \mu$ and $\text{Var}[X_i] = \sigma^2 < \infty$, and let
\[S_n = \frac{X_1 + X_2 + \cdots + X_n}{n}
      = \frac{1}{n}\sum_{i}^{n} X_i\]
denote their mean. Then as $n$ approaches infinity, the random variables $\sqrt{n}(S_n - \mu)$ converge in distribution to a normal $\mathcal{N}(0, \sigma^2)$.


\subsection{How to create Sections and Subsections}

Use section and subsections to organize your document. Simply use the section and subsection buttons in the toolbar to create them, and we'll handle all the formatting and numbering automatically.

\subsection{How to add Lists}

You can make lists with automatic numbering \dots

\begin{enumerate}
\item Like this,
\item and like this.
\end{enumerate}
\dots or bullet points \dots
\begin{itemize}
\item Like this,
\item and like this.
\end{itemize}

\subsection{How to add Citations and a References List}

You can upload a \verb|.bib| file containing your BibTeX entries, created with JabRef; or import your \href{https://www.overleaf.com/blog/184}{Mendeley}, CiteULike or Zotero library as a \verb|.bib| file. You can then cite entries from it, like this: \cite{greenwade93}. Just remember to specify a bibliography style, as well as the filename of the \verb|.bib|.

You can find a \href{https://www.overleaf.com/help/97-how-to-include-a-bibliography-using-bibtex}{video tutorial here} to learn more about BibTeX.

We hope you find Overleaf useful, and please let us know if you have any feedback using the help menu above --- or use the contact form at \url{https://www.overleaf.com/contact}!

\bibliographystyle{alpha}
\bibliography{sample}

\end{document}