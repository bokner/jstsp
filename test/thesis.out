\BOOKMARK [1][-]{section.1}{\376\377\004\037\004\076\004\101\004\102\004\060\004\075\004\076\004\062\004\072\004\060\000\040\004\067\004\060\004\064\004\060\004\107\004\070}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\004\037\004\100\004\070\004\074\004\065\004\100}{}% 2
\BOOKMARK [1][-]{section.3}{\376\377\000\050}{}% 3
\BOOKMARK [2][-]{subsection.3.1}{\376\377\000H\000o\000w\000\040\000t\000o\000\040\000a\000d\000d\000\040\000C\000o\000m\000m\000e\000n\000t\000s}{section.3}% 4
\BOOKMARK [2][-]{subsection.3.2}{\376\377\000H\000o\000w\000\040\000t\000o\000\040\000a\000d\000d\000\040\000T\000a\000b\000l\000e\000s}{section.3}% 5
\BOOKMARK [2][-]{subsection.3.3}{\376\377\000H\000o\000w\000\040\000t\000o\000\040\000w\000r\000i\000t\000e\000\040\000M\000a\000t\000h\000e\000m\000a\000t\000i\000c\000s}{section.3}% 6
\BOOKMARK [2][-]{subsection.3.4}{\376\377\000H\000o\000w\000\040\000t\000o\000\040\000c\000r\000e\000a\000t\000e\000\040\000S\000e\000c\000t\000i\000o\000n\000s\000\040\000a\000n\000d\000\040\000S\000u\000b\000s\000e\000c\000t\000i\000o\000n\000s}{section.3}% 7
\BOOKMARK [2][-]{subsection.3.5}{\376\377\000H\000o\000w\000\040\000t\000o\000\040\000a\000d\000d\000\040\000L\000i\000s\000t\000s}{section.3}% 8
\BOOKMARK [2][-]{subsection.3.6}{\376\377\000H\000o\000w\000\040\000t\000o\000\040\000a\000d\000d\000\040\000C\000i\000t\000a\000t\000i\000o\000n\000s\000\040\000a\000n\000d\000\040\000a\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e\000s\000\040\000L\000i\000s\000t}{section.3}% 9
